docassemble/MAPetitionToSealEviction/data/questions/petition_to_seal_eviction.yml

```
@@ -432,9 +427,9 @@ id: Fault Eviction
 question: |
   Do you have any other recent eviction cases?
 subquestion: |
-  The judge needs to know about evictions in the last **7 years**.
+  Has a landlord filed a new eviction case against you at any time in the **LAST 7 YEARS?**
```

```suggestion
  Has a landlord filed a new eviction case against you at any time in the **last 7 years?**
```

We never use all caps for emphasis. See: https://assemblyline.suffolklitlab.org/docs/style_guide/formatting#only-use-all-capital-letters-for-acronyms-and-initialisms